packages/web-socket-browser/src/ws-browser-factory.ts

```diff
@@ -5,15 +5,17 @@ import { JSONParserV1 } from 'json-helpers';
 import { uuidProvider } from './uuid';
 import { createWebSocketClient as createThin } from './ws-browser-factory-thin';
 
+import type { ThinContext} from './ws-browser-factory-thin';
 import type { IpcBusClient } from '@electron-common-ipc/universal';
 
-export function createWebSocketClient(): IpcBusClient {
+export function createWebSocketClient(ctx?: Partial<ThinContext>): IpcBusClient {
     const container = new GlobalContainer();
 
     return createThin({
         uuidProvider,
```

Comment:
since we passed ctx to this function we should try applying to every dependency passed to `createThin` like:
`uuidProvider: ctx?.uuidProvider ?? uuidProvider`